glean-core/src/upload/directory.rs

```diff
@@ -267,18 +288,20 @@ impl PingDirectoryManager {
     /// # Returns
     ///
     /// A vector of tuples with the file size and payload of each ping file in the directory.
```

Comment:
Mind updating the doc-comment for the return type to show it now also returns the `discarded_files`?